audio/Makefile

```diff
@@ -49,6 +49,8 @@ ifeq ($(CONFIG_AUDIO_FORMAT_PCM),y)
   CSRCS += pcm_decode.c
 endif
 
+include libsrc/Make.defs
```

Comment:
Isn't that the other way around? 
i.e. libsrc/Make.defs should include audio/Makefile.